test/verify/test_gemm_transposea.cpp

```diff
@@ -46,3 +46,6 @@ struct test_gemm_transposea : verify_program<test_gemm_transposea<DType>>
 template struct test_gemm_transposea<migraphx::shape::float_type>;
 template struct test_gemm_transposea<migraphx::shape::half_type>;
 template struct test_gemm_transposea<migraphx::shape::fp8e4m3fnuz_type>;
+// TODO need hipblaslt support
```

Comment:
TODO make sense. Make this a separate issue for tracking

Reply:
https://github.com/ROCm/AMDMIGraphX/issues/3419